manifests/client.pp

```diff
@@ -97,10 +97,12 @@
     notify  => Service[$agent_service_name],
   }
 
+  Concat::Fragment {
+    target => 'ossec.conf',
+    notify => Service[$agent_service_name]
+  }
+
```

Comment:
as opposed to the default resource title, which is contained to a single block,
this might set default values for other concat::fragment resources as well and can have unindented side effects.

https://docs.puppet.com/puppet/3.8/lang_defaults.html
vs
https://docs.puppet.com/puppet/4.10/lang_resources_advanced.html#per-expression-default-attributes